src/xml.c

```diff
@@ -215,6 +215,11 @@ lyxml_dup_elem(struct ly_ctx *ctx, struct lyxml_elem *elem, struct lyxml_elem *p
         if (parent) {
             lyxml_add_child(ctx, parent, dup);
         }
+        else if(result) {
```

Comment:
Please put `else` on the same line as preceding `}` and put a space after `if`, thanks.